recipes/mender-orchestrator/debian-master/rules

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@ --with systemd
```

Comment:
I guess this is taken from `mender-client4`? What is it for? I'd expect things like `--with systemd` to be done in `configure`.